PTables_Doris1115.sql

```diff
@@ -0,0 +1,62 @@
+USE Proj_B1
+ /*
+3.1 Populate the following look-up tables: tblMONTH
+  */
+CREATE PROCEDURE addNewMonth
+@MonthName Varchar(10),
+@MonthNum INT
+AS
+BEGIN TRANSACTION A3
+INSERT INTO tblMONTH (MonthName, MonthNum) VALUES (@MonthName, @MonthNum)
+IF @@error <> 0
+    ROLLBACK TRANSACTION
+ELSE
+    COMMIT TRANSACTION
```

Comment:
Rather then do a procedure, just create the code for a simple insert 5 times for each of the populates. ie. INSERT INTO () VALUES (valueSet1), (valueSet2), etc